srcpkgs/libgdal/template

```diff
@@ -46,3 +82,17 @@ libgdal-devel_package() {
 		vmove usr/lib/*.so
 	}
 }
+
+python3-gdal_package() {
+	depends="${sourcepkg}>=${version}_${revision}"
+	short_desc+=" - Python3 bindings"
+	pkg_install() {
+		vmove usr/bin/*.py
+		vmove "usr/lib/python*"
+		vlicense $wrksrc/$build_wrksrc/LICENSE.TXT
+		vdoc $wrksrc/$build_wrksrc/swig/python/README.rst
+		mkdir -p ${PKGDESTDIR}/usr/share/${pkgname}
```

Comment:
```suggestion
		vmkdir usr/share/python3-gdal
```
(using ${pkgname} isn't wrong, but writing out is imo more clear, and pkgnames usually don't change that often)q